package.json

```diff
@@ -119,7 +119,6 @@
     "puppeteer": "~10.0.0",
     "rollup": "2.42.3",
     "rollup-plugin-sourcemaps": "^0.6.3",
-    "semiver": "^1.1.0",
```

Comment:
I removed this as a dependency since we were only using it in a few places and already depending on `semver`.

I noticed references to this package in `scripts/license.ts` and `NOTICE.md` — is there any manual intervention needed to sort those out?

Reply:
possible nvm on this one - found `scripts/license.ts` and ran it, see https://github.com/ionic-team/stencil/pull/3346/commits/684fe50b2b11a89a890744e1961df77897f5dd8d